### DISCUSSION TITLE
Problems setting up rtorrent inject

### DISCUSSION THREAD

**Discussion by mrwsl** [cross-seed/cross-seed#359] [Q&A] [answered]
Hi there,
I'm trying to use the inject method with my [swizzin](https://swizzin.ltd/applications/rtorrent#rpc-mounts) based installation of r(u)torrent.

Whenever I try to run `cross-seed search` this error comes up:
```
error: Failed to reach rTorrent at https://user:[REDACTED]@localhost:443/rutorrent/plugins/httprpc/action.php
```

The same endpoint works for radarr and sonarr. What am I missing? Thanks :)

**Comment by DoAndroids** [comment, marked as answer]
You sure you're using https + "localhost" ?
Mine is https://user:[REDACTED]@domain.com/rtorrent/rpc

**Reply by mrwsl** [reply]
Thanks! A bit weird since `localhost` is working for radarr and sonarr.

**Reply by DoAndroids** [reply]
Yeah, but possibly using "http" rather than https?

**Reply by mrwsl** [reply]
The settings say it uses SSL.

**Reply by DoAndroids** [reply]
Hmm, I wonder if Radarr / Sonarr drops the SSL setting if it detects "localhost"...
Either way, at least you have it working now I hope 🙂

**Comment by Go0ners** [comment]
Hello @herrwusel can you tell me how you solved the problem? on my side impossible to connect r(u)torrent.

**Reply by mrwsl** [reply]
I use the tld instead of localhost.

**Reply by Go0ners** [reply]
If you want, you can make a copy of your configuration?

**Reply by mrwsl** [reply]
Sure:
```
rtorrentRpcUrl: "https://user:password@example.com/rutorrent/plugins/httprpc/action.php",
```
Remember that your endpoint to rpc might be different. Mine is set up differently from default iirc.
